api/src/main/java/com/velocitypowered/api/proxy/Player.java

```diff
@@ -45,6 +46,21 @@
    */
   String getUsername();
 
+  /**
+   * The locale the velocity proxy is translating its messages to. By default {@link PlayerSettings#getLocale()}
+   *
+   * <p>Can be {@code null} when the settings have not been initialized yet.</p>
```

Comment:
This is still imprecise - the wording I originally suggested is better here.

Reply:
Oh, I overlooked that you did other changes than the code null thing